### ISSUE TITLE
Hyper renders leftover text after using clear/vim/nano from prior render

### ISSUE THREAD

**Issue by deuks** [zeit/hyper#3792]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [?] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10
- **Hyper.app version**: 3.0.2
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/deuks/329e4f517ce1b930293df7ce8ddf1100

- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: I have a picture from below using vanilla. It also occurs with plugins

## Issue
I feel like this issue has kind of been addressed before, and I tried those troubleshooting steps but did not have any luck.  

I use hyper to ssh into a linux vm. Any time I vim a file, or try to clear my active window, part of the shell prior to the one rendered now is still rendered.  I tried with and without plugins, I did try a couple different configuration under the env: {} parameter as other posts suggested.  Either I have improper syntax or something else is still wrong.  

In my picture, you can see the after I do a clear statement, that last line is still appearing after everything else clears.  If I scroll up, and then back down, that line would disappear and everything would be fine.  It's just annoying to scroll up and scroll down to get the correct view

![Screen Shot 08-15-19 at 12 44 PM](https://user-images.githubusercontent.com/49660052/63122988-a3aee780-bf6d-11e9-8962-f1cfc700e6fe.JPG)

**Comment by kczx3** [comment]
I've noticed this every time that I have to use Hyper.  I download it and try it every few months or so.  Its always the same when using vim.  Its really not a usable app if it can't render vim correctly.

**Comment by Melanpan** [comment]
> 
> 
> I've noticed this every time that I have used Hyper. I download it and try it every few months or so. Its always the same when using vim. Its really not a usable app if it can't render vim correctly.

Yup, this has been an issue since the release of Hyper 3 which was in May! I can't wrap my head around them not even fixing this issue, making hyper unusable for a lot of Windows users.

**Comment by dontsave** [comment]
Please fix this issue!

**Comment by benjaminpottier** [comment]
Seems like this problem still exists. Any update on a fix?
